src/test/java/io/micrometer/newrelic/NewRelicRegistryTest.java

```diff
@@ -117,8 +118,9 @@ void testPublishTimeGauge() {
             value -> 5);
 
     newRelicRegistry.publish();
-
-    verify(newRelicSender).sendBatch(expectedBatch);
+    // TODO Replace with verify(newRelicSender).sendBatch(expectedBatch) when SDK dependecy is fixed
+    //  Possibly do this with other tests
+    verify(newRelicSender).sendBatch(Mockito.refEq(expectedBatch));
```

Comment:
we need to make sure that the expectedBatch has been constructed with a different attributes instance than the one we're passing into the sender. Because it's mutable, we're unable to verify anything when we're using the same one for both cases.

Reply:
Sorry. I wasn't asking for docs. I think the test needs to be updated so that we are actually testing that we're adding the attributes requested.

Reply:
I took a look, and this now looks great. Thanks!